src/riak_index.erl

```diff
@@ -360,8 +360,6 @@ apply_continuation(Q, C) ->
 %% @doc upgrade a query to the current latest version
```

Comment:
Why is upgrade/downgrade stuff in `riak_index.erl`? I know you are only updating what is already here.  Thought this is for 2i.